frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/StartHandler.java

```diff
@@ -188,19 +186,48 @@ public APIGatewayProxyResponseEvent handleRequest(
             LOG.info("previousSessionId: {}", previousSessionId);
             authSessionService.addOrUpdateSessionId(previousSessionId, session.getSessionId());
 
+            var clientSessionId =
+                    getHeaderValueFromHeaders(
+                            input.getHeaders(),
+                            CLIENT_SESSION_ID_HEADER,
+                            configurationService.getHeadersCaseInsensitive());
+            var txmaAuditHeader =
+                    getOptionalHeaderValueFromHeaders(
+                            input.getHeaders(), TXMA_AUDIT_ENCODED_HEADER, false);
+            String internalCommonSubjectIdentifierForAuditEvent = AuditService.UNKNOWN;
+            String phoneNumber = AuditService.UNKNOWN;
+
+            var auditContext =
+                    new AuditContext(
+                            userContext.getClient().get().getClientID(),
+                            clientSessionId,
+                            session.getSessionId(),
+                            internalCommonSubjectIdentifierForAuditEvent,
+                            userContext
+                                    .getUserProfile()
+                                    .map(UserProfile::getEmail)
+                                    .orElse(AuditService.UNKNOWN),
+                            IpAddressHelper.extractIpAddress(input),
+                            phoneNumber,
+                            extractPersistentIdFromHeaders(input.getHeaders()),
+                            txmaAuditHeader);
+
+            boolean isBlockedForReauth = false;
+            if (configurationService.isAuthenticationAttemptsServiceEnabled() && reauthenticate) {
+                isBlockedForReauth =
+                        checkUserIsBlockedForReauthAndEmitFailureAuditEvent(
```

Comment:
This event is being emitted before the `AUTH_REAUTH_REQUESTED` event at line 259 which makes the order of the events difficult to follow.  Consider moving the code that emits the `AUTH_REAUTH_REQUESTED` to a point before this event is emitted.